src/stylesheets/common/_documentation.scss

```diff
@@ -7,12 +7,29 @@
   align-items: center;
   justify-content: center;
   padding: 40px 20px;
-
   h1 {
+    
+  }
+  .documentation-header{
+    /*former .documentation-hero h1 style*/
     padding-top: 20px;
     color: #fff;
+    /* _documentation.scss style for h1*/
+    position: relative;
+    font-weight: 700;
```

Comment:
Several of these should be already inherited, right?  No need to include those here.